ports/openttd/gameinfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version='1.0' encoding='utf-8'?>
+<gameList>
+  <game>
+    <path>./OpenTTD.sh</path>
+    <name>OpenTTD</name>
+    <desc>OpenTTD is an open source simulation game based upon Transport Tycoon Deluxe which was originally released in November of 1994.</desc>
+    <image>./openttd/screenshot.png</image>
+    <releasedate>19941101T000000</releasedate>
+    <developer>Chris Sawyer</developer>
```

Comment:
The Original  TTD is from Chris but not the OpenSource game. Since it's not even using assets from the original game they don't have anything in common ^^

Reply:
Ah.  Of course.  Lemme fix that.